tests/by-util/test_base32.rs

```diff
@@ -88,10 +88,7 @@ fn test_wrap_no_arg() {
         let ts = TestScenario::new(util_name!());
         let expected_stderr = &format!(
```

Comment:
could you please fix the clippy warning?
`ERROR: `cargo clippy`: useless use of `format!` (file:'/home/runner/work/coreutils/coreutils/tests/by-util/test_base32.rs', line:89)`

Reply:
Fixed